src/orders/OrderProcessor.sol

```diff
@@ -18,6 +18,7 @@ import {IdShare} from "../IdShare.sol";
 import {FeeLib} from "../common/FeeLib.sol";
 import {IForwarder} from "../forwarder/IForwarder.sol";
 import {IFeeSchedule} from "./IFeeSchedule.sol";
+import {IVaultRouter} from "../IVaultRouter.sol";
```

Comment:
Remove all changes to OrderProcessor. The fulfiller should widraw from the vault, then call fulfill